python-test/features/steps/agentsProvider.py

```diff
@@ -0,0 +1,55 @@
+from behave import given, when, then
+import time
+import functions
+
+base_orb_url = "https://beta.orb.live"
```

Comment:
I'm not sure how variables work in behave, but it seems like this should be pulled out somewhere?

Reply:
you are right. Maybe as an environment variable, just like email and password. I'll talk to Stefano about the best way to do it.